### PR TITLE
[RPC] add new utxoupdatepsbt arguments to the CRPCCommand and CPRCCvertParam tables

### PR DESCRIPTION
**Pull request by jnewbery**
The new `descriptors` argument was not added to the CRPCCommand and CPRCCvertParam tables, meaning that it couldn't be used with bitcoin-cli or named arguments.

Before this PR:

```
> bitcoin-cli utxoupdatepsbt 'cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==' "[{\"desc\":\"sh(wpkh([bd50871a/0h/0h/0h]03895c66337b38699bfafff1084ad35bc347fac4f4e5e5fe5eb7dd81155280db53))\"}]"
error code: -3
error message:
Expected type array, got string
> bitcoin-cli --named utxoupdatepsbt psbt='cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==' descriptors="[{\"desc\":\"sh(wpkh([bd50871a/0h/0h/0h]03895c66337b38699bfafff1084ad35bc347fac4f4e5e5fe5eb7dd81155280db53))\"}]"
error code: -8
error message:
Unknown named parameter descriptors
```

After this PR:

```
bitcoin-cli utxoupdatepsbt 'cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==' "[{\"desc\":\"sh(wpkh([bd50871a/0h/0h/0h]03895c66337b38699bfafff1084ad35bc347fac4f4e5e5fe5eb7dd81155280db53))\"}]"
cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==
bitcoin-cli --named utxoupdatepsbt psbt='cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==' descriptors="[{\"desc\":\"sh(wpkh([bd50871a/0h/0h/0h]03895c66337b38699bfafff1084ad35bc347fac4f4e5e5fe5eb7dd81155280db53))\"}]"
cHNidP8BAFMCAAAAAYCdwVRx2X3o4KHx5tAMsN1ddp51MbfWsietjfMbl5HtAAAAAAD/////AQDh9QUAAAAAF6kUW+rtEOi4nk9rpw2F5XZl1dd8ehGHAAAAAAAAAA==
```

### CODE REVIEW

**Comment by jnewbery** [comment]
Follow-up to #15427. Reported by @ariard here: https://github.com/bitcoin/bitcoin/pull/15427#pullrequestreview-239730895.

**Comment by sipa** [comment]
Concept ACK, code looks goods.

**Comment by MarcoFalke** [comment]
Could make one of the calls in the functional tests use named args?

**Comment by MarcoFalke** [comment]
Does  #15427 need release notes? If so, you may add them in this pull.

**Review by fanquake — APPROVED** [review]
ACK df699195f4866068b86acf914b0aed8cc8f91e49

Checked that this fixes `bitcoin-cli` and named arguments usage.

@jnewbery did you want to update the test or add release notes here?

**Comment by jnewbery** [comment]
Added named arguments to the test and release notes.

File: doc/release-notes-15427.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Updated RPCs
+------------
+
+The `utxoupdatepsbt` RPB method has been updated to take a `descriptors`
```
Comment (fanquake) [inline doc/release-notes-15427.md]: nit: RPC

**Review by fanquake — APPROVED** [review]
ACK a52ec281b0ceed278c8b65524dc1ab601281ce27

Thanks for updating the test and adding the release notes for #15427.

File: doc/release-notes-15427.md (hunk 1 above)
Comment (jnewbery) [reply doc/release-notes-15427.md]: thanks. Fixed

**Review by fanquake — APPROVED** [review]
re-ACK 91cc18f602fe2ff7fe47335a8e1e7734895a19d9

**Comment by promag** [comment]
ACK 91cc18f.

**Pull request merged**